src/picongpu/include/particles/ionization/byField/ADK/AlgorithmADK.hpp

```diff
@@ -82,8 +85,21 @@ namespace ionization
                 float_X dBase = float_X(4.0) * util::cube(protonNumber) / (eInAU * util::quad(nEff)) ;
                 float_X dFromADK = math::pow(dBase,nEff);
 
+                float_X polarizationFactor;
+                /* set up the polarization factor */
+                if(T_linPol)
```

Comment:
:+1: Yep, looks good to me. I think it's okay to stay away from overcomplicating things and not introduce a new functor for this.